tests/test_utils_display.py

```diff
@@ -1,9 +1,18 @@
 from io import StringIO
-from unittest import TestCase, mock
+from unittest import TestCase, mock, skipIf
 
 from scrapy.utils.display import pformat, pprint
 
+try:
+    import bpython
 
+    bpy = True
+    del bpython
+except ImportError:
+    bpy = False
+
+
+@skipIf(not bpy, "bpython not available in testenv")
```

Comment:
Why is this needed? Looks like this test works without bpython and doesn't use it?

Reply:
Without this, we have errors on this tests:

![image](https://github.com/scrapy/scrapy/assets/144740771/fd1b1a69-f2a1-44b3-b841-2b03a46dec6a)

Like a said on this comment: https://github.com/scrapy/scrapy/pull/6272#discussion_r1518409690

Do you think there is a better way?